specification/maps/data-plane/Route/preview/2023-08-01-preview/route.json

```diff
@@ -0,0 +1,1807 @@
+              {
+                "value": "USHazmatClass2",
+                "description": "Compressed gas"
+              },
+              {
+                "value": "USHazmatClass3",
+                "description": "Flammable liquids"
+              },
+              {
+                "value": "USHazmatClass4",
+                "description": "Flammable solids"
+              },
+              {
+                "value": "USHazmatClass5",
+                "description": "Oxidizers"
+              },
+              {
+                "value": "USHazmatClass6",
+                "description": "Poisons"
+              },
+              {
+                "value": "USHazmatClass7",
+                "description": "Radioactive"
+              },
+              {
+                "value": "USHazmatClass8",
+                "description": "Corrosives"
+              },
+              {
+                "value": "USHazmatClass9",
+                "description": "Miscellaneous"
+              },
+              {
+                "value": "otherHazmatExplosive",
+                "description": "Explosives"
+              },
+              {
+                "value": "otherHazmatGeneral",
+                "description": "Miscellaneous"
+              },
+              {
+                "value": "otherHazmatHarmfulToWater",
+                "description": "Harmful to water"
+              }
+            ]
+          }
+        }
+      }
+    },
+    "NextLink": {
+      "type": "string",
+      "description": "The is the link to the next page of the features returned. If it's the last page, no this field."
+    },
+    "OptionalId": {
+      "type": "object",
+      "description": "optional Id",
+      "properties": {
+        "optionalId": {
+          "description": "id of the request which would show in corresponding batchItem",
+          "type": "string"
+        }
+      }
+    },
+    "DirectionsBatchRequestItem": {
+      "description": "Batch Query object",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/OptionalId"
+        },
+        {
+          "$ref": "#/definitions/DirectionsRequest"
+        }
+      ]
+    },
+    "DirectionsBatchRequestBody": {
+      "description": "The list of directions queries/requests to process. The list can contain a max of 100 queries and must contain at least 1 query.",
+      "type": "object",
+      "properties": {
+        "batchItems": {
+          "description": "The list of queries to process.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DirectionsBatchRequestItem"
+          }
+        }
+      }
+    },
+    "DirectionsBatchResponseItem": {
+      "description": "Batch response item",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/OptionalId"
+        },
+        {
+          "$ref": "#/definitions/RouteDirectionsResponse"
+        },
+        {
+          "type": "object",
```

Comment:
for users wanting to check if a particular item was successful or not, empty 'error' field is fine as a convention but consider adding a status field to items to be more explicit.